app/screens/scheduled_post_options/footer/scheduled_post_footer.tsx

```diff
@@ -2,17 +2,18 @@
 // See LICENSE.txt for license information.
 
 import {BottomSheetFooter, type BottomSheetFooterProps} from '@gorhom/bottom-sheet';
+import {Button} from '@rneui/base';
```

Comment:
We have a button component in `@components/button`, that mostly work on top of `@rneui/base` button. Any reason not to use that one? It should simplify very much the code.

Reply:
Styling it with full width was causing a lot of issues, so I moved on to this button. It also had issues with icon's alignment, and fixing it required updates wherever this particular icon was being used.